src/models/informer.py

```diff
@@ -3,3 +3,439 @@
 
 class Informer(TorchModel, config_name='informer'):
     pass
+
+class GaussianBasisFunctions(object):
```

Comment:
We don't need t write object, we don't  use Python2
```suggestion
class GaussianBasisFunctions:
```